### PR TITLE
Add privacy link on manage students page

### PR DESCRIPTION
**Pull request by epeach**
Added a link to the privacy handout on the manage student teacher dashboard page.

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/teacher-dashboard/section_manage.haml [hunk 1]
```diff
@@ -172,3 +172,7 @@ content-type: text/ng-template
 
   #login-type-react{'ng-init' => '$emit("login-type-react-rendered");'}
   #share-setting-react{'ng-init' => '$emit("share-setting-react-rendered");'}
+
+  %br/
+  %div{id: 'privacy_link'}
+    =I18n.t('dashboard_privacy_document_link')
```
Comment (caleybrock) [inline pegasus/sites.v3/code.org/public/teacher-dashboard/section_manage.haml]: Looks like the string is: `<a href="/privacy/student-privacy", target="_blank">Click here</a> for our privacy document that can be shared or printed for your students.`

I remember @Bjvanminnen saying we don't love adding html into strings, although there seem to be lots of html in our string file. Should be be substituting in variables here?

Comment (Bjvanminnen) [reply pegasus/sites.v3/code.org/public/teacher-dashboard/section_manage.haml]: Yeah, that's not good. I'm also a little surprised you can just embed a string like that in haml and have it work

**Comment by epeach** [comment]
@Bjvanminnen PTAL.

File: pegasus/sites.v3/code.org/public/teacher-dashboard/section_manage.haml [hunk 2]
```diff
@@ -172,3 +172,9 @@ content-type: text/ng-template
 
   #login-type-react{'ng-init' => '$emit("login-type-react-rendered");'}
   #share-setting-react{'ng-init' => '$emit("share-setting-react-rendered");'}
+
+  %br/
+  %div{id: 'privacy_link'}
+    %a{'ng-href' => CDO.code_org_url("/privacy/student-privacy"), target: "_blank"}
+      =I18n.t('click_here')
+    =I18n.t('privacy_doc_explanation')
```
Comment (Bjvanminnen) [inline pegasus/sites.v3/code.org/public/teacher-dashboard/section_manage.haml]: Technically this is much better/safer. However, I would check with PM, as I'm not sure this structure will translate well (i.e. when translating `privacy_doc_explanation` I don think translators necessarily get the context, which is tough for a sentence fragment like this.

I wonder if we should just have the entire sentence be a link.

**Comment by epeach** [comment]
@Bjvanminnen PTAL - I checked with Poorva and made the change you suggested.

**Pull request merged**